2012-04-03  Jeff Law  <law@redhat.com>

	* locales/lv_LV: Fix first_weekday and first_workday.

diff -rup c/localedata/locales/lv_LV d/localedata/locales/lv_LV
--- c/localedata/locales/lv_LV	2012-01-01 05:16:32.000000000 -0700
+++ d/localedata/locales/lv_LV	2012-04-03 11:13:40.590928949 -0600
@@ -2183,6 +2183,14 @@ t_fmt_ampm  ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+
+% Define the first day of the week to be displayed in a calendar.
+% This weekday is relative to the date defined in the <week> keyword.
+first_weekday 2
+
+% Define the first workday relative to the <week> keyword
+first_workday 2
+
 END LC_TIME
 
 LC_MESSAGES
